sketch.js

```diff
@@ -2,6 +2,8 @@ function sketch_header_background(p) {
 
   // p5 setup here
   p.setup = function () {
+    p.frameRate(15);
+
```

Comment:
Lower framerate, such a small and visually-simple animation still looks smooth to the human eye.